src/pages/brain/[pageSlug].tsx

```diff
@@ -0,0 +1,99 @@
+import { Center } from "@chakra-ui/react"
+import CustomImage from "components/brain/CustomImage"
+import CustomLink from "components/brain/CustomLink"
+import Header from "components/brain/Header"
+import PageBrainCard from "components/brain/PageBrainCard"
+import {
+  getAllPages,
+  getRelatedPageLinks,
+} from "components/brain/utils/brainFetchers"
+import Layout from "components/common/Layout"
+import LinkPreviewHead from "components/common/LinkPreviewHead"
+import CategorySection from "components/explorer/CategorySection"
+import { GetServerSideProps } from "next"
+import Image from "next/image"
+import { NotionAPI } from "notion-client"
+import { NotionRenderer } from "react-notion-x"
+import "react-notion-x/src/styles.css"
+import slugify from "slugify"
+
+const PageDetails = ({ blockMap, linkedPageContents, pageId, pageLogo }) => (
+  <>
+    <LinkPreviewHead path="" />
+    <Layout
+      backButton={{ href: "/guildverse", text: "Go back to Guildverse" }}
+      title={blockMap.block[pageId]?.value.properties.title[0][0]}
+      image={
+        pageLogo && (
+          <Center boxSize={16} position="relative">
+            <Image
+              src={pageLogo}
+              layout="fill"
+              objectFit="contain"
+              quality="10"
+              style={{
+                overflow: "visible",
+              }}
+              alt="logo"
+            ></Image>
+          </Center>
+        )
+      }
+    >
+      <NotionRenderer
+        recordMap={blockMap}
+        forceCustomImages={true}
+        components={{
+          nextImage: CustomImage,
+          Collection: Header,
+          PageLink: CustomLink,
+        }}
+      />
+      {linkedPageContents && (
+        <CategorySection fallbackText={"there are no linked pages"} mt="24px">
```

Comment:
You could use Chakra sizes here, so we stay consistent. 24px would be `6`. I suggest you to always try to pick a pre-defined size from our style config, and only use px values if you really need to fine-tune things. ;)

Also, the curly brackets are unnecessary around the `fallbackText` prop and please start the title with uppercase letter.